<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFilterBarButtons</class>
 <widget class="QFrame" name="CFilterBarButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>23</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filter bar buttons</string>
  </property>
  <layout class="QHBoxLayout" name="hl_filterBarButtons">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="le_Count">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Number of entries</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tb_Filter">
     <property name="toolTip">
      <string>filter</string>
     </property>
     <property name="text">
      <string>filter</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/application-search-result.png</normaloff>:/diagona/icons/diagona/icons/application-search-result.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tb_RemoveFilter">
     <property name="toolTip">
      <string>remove filter</string>
     </property>
     <property name="text">
      <string>remove filter</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/cross-circle.png</normaloff>:/diagona/icons/diagona/icons/cross-circle.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tb_ClearForm">
     <property name="toolTip">
      <string>clear form</string>
     </property>
     <property name="text">
      <string>clear form</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tb_New">
     <property name="toolTip">
      <string>new (clear + remove filter)</string>
     </property>
     <property name="text">
      <string>new</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/cross-button.png</normaloff>:/diagona/icons/diagona/icons/cross-button.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="hs_FilterBarButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
